src/Microsoft.AspNetCore.SignalR.Specification.Tests/RedisHubLifetimeManagerTests.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
These should continue living in the test folder and just use the `Specification.Tests` project